ProcessList.c

```diff
@@ -142,20 +142,21 @@ static void ProcessList_buildTree(ProcessList* this, pid_t pid, int level, int i
    Vector_delete(children);
 }
 
+static long ProcessList_treeProcessCompare(const void* v1, const void* v2) {
+   const Process *p1 = (const Process*)v1;
+   const Process *p2 = (const Process*)v2;
+
+   return p1->pid - p2->pid;
+}
+
 void ProcessList_sort(ProcessList* this) {
    if (!this->settings->treeView) {
       Vector_insertionSort(this->processes);
```

Comment:
Couldn't `Vector_quickSortCustomCompare` be also used for normal sorting (given the right comparison function)?

Reply:
`Vector_quickSort` uses the *right* one?

Reply:
Just wondered if we have code duplication there.